ooni/ui/cli.py

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
Yes correct. Everything inside of ooni/script/* is also an actual script that you can run indipendently and it will behave as a standard python script with an entry point called `run()`.

What is inside of `ooni/foo/cli.py` are the functions related to supporting the execution of the script.